test/functional/test_framework/address.py

```diff
@@ -21,11 +21,15 @@
     taproot_construct,
 )
 from .util import assert_equal
+from test_framework.script_util import (
+    keyhash_to_p2pkh_script,
+    program_to_witness_script,
+    scripthash_to_p2sh_script,
+)
 from test_framework.segwit_addr import (
     decode_segwit_address,
     encode_segwit_address,
 )
-
```

Comment:
stylistic nit: we usually keep two blank lines after imports, so should rather add another one than remove one here (see also https://peps.python.org/pep-0008/#blank-lines).

Reply:
Thanks :100: